tests/unit/mixins/data-adapter-mixin-test.js

```diff
@@ -39,6 +39,13 @@ describe('DataAdapterMixin', () => {
       expect(hash).to.have.ownProperty('beforeSend');
     });
 
+    it('asserts the presence of authorizer', () => {
+      adapter.set('authorizer', null);
+      expect(function() {
+        adapter.ajaxOptions();
+      }).to.throw(/You\'re using the DataAdapterMixin without specifying an authorizer. Please add `authorizer: \'authorizer:application\'` to your adapter\./);
```

Comment:
Just asserting that it throws is fine, otherwise the test has to be updated whenever the wording of the assertion message changes.